pkg/cmd/server/server.go

```diff
@@ -463,6 +464,7 @@ func (s *server) initRestic(config api.ObjectStorageProviderConfig) error {
 		os.Setenv("AZURE_ACCOUNT_KEY", os.Getenv("AZURE_STORAGE_KEY"))
 	}
 
+	// TODO do we want to use a kube informer factory?
```

Comment:
We could, although all informers in the factory have to share the same options (`FieldSelector`, in this case). I'm not sure it buys us much?

Reply:
Yeah, I was thinking it probably didn't make sense to, but wanted to think about it